src/ClientData/include/ClientData/TracepointInfo.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_TRACEPOINT_INFO_H_
+#define CLIENT_DATA_TRACEPOINT_INFO_H_
+
+#include <string>
+#include <utility>
+
+namespace orbit_client_data {
+
+// This class is used on the client to represent a unique tracepoint, containing the tracepoint's
+// name and category.
+class TracepointInfo {
+ public:
+  TracepointInfo() = delete;
+  TracepointInfo(std::string name, std::string category)
```

Comment:
I think we should invert the parameters to reflect that the category usually comes before the name, e.g., `sched:sched_switch` -> `TracepointInfo{"sched", "sched_switch"}`, not `TracepointInfo{"sched_switch", "sched"}`.

Reply:
Same but less importantly with the order of fields and accessors.

Reply:
Done.